### PR TITLE
499 org root part1 navbar

### PR DESCRIPTION
**Pull request by Developer3027**
# 🔗 Issue
[issue 4++](https://github.com/rubyforgood/pet-rescue/issues/499)

# ✍️ Description
Reworked original navbar for org/root to comply with updated Figma file.

# 📷 Screenshots/Demos

Screenshot from 2024-03-06 17-09-19
Screenshot from 2024-03-06 17-12-10
Screenshot from 2024-03-06 17-14-03

### CODE REVIEW

**Comment by Developer3027** [comment]
The application.html.erb file has note for review of new navbar. Will be removed later once page is created and routing is correct.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <body>
     <%= render "layouts/shared/flash_messages" %>
     <% if Current.tenant.present? %>
+      <!-- //? set to org_navbar to review ?// -->
```
Comment (kasugaijin) [inline app/views/layouts/application.html.erb:8]: Would be good to keep the same naming and just overwrite the current "layouts/shared/navbar" with your new navbar logic?

Comment (Developer3027) [reply app/views/layouts/application.html.erb:8]: Sounds good. I will drop the original navbar and replace with v2.

Comment (Developer3027) [reply app/views/layouts/application.html.erb]: navbar updated.
org-navbar deleted.
application.html.erb note removed
pushed

**Review by kasugaijin — APPROVED** [review]
LGTM

**Pull request merged**